add-on/panels/css/panel.css

```diff
@@ -207,15 +195,39 @@ body {
 }
 
 .room-list {
-  /* xxx  not sure why flex needs the 3 value setting
-    but setting flex to just 1, the whole tab including the new room is scrollable.
-    seems to not like the 0% of the default setting - may be FF bug */
-  flex: 1 1 0;
   overflow-y: auto;
   overflow-x: hidden;
-  display: flex;
   flex-flow: column nowrap;
   width: 100%;
+  /* max-height is determined by the max number of rooms shown,
+   * which are 5 @ 28px each + padding */
+  max-height: 144px;
+  min-height: 53px;
+  padding-top: 4px;
+}
+
+.room-list-empty {
+  display: inline-block;
+  min-height: 80px;
+}
+
+
+/* Adds blur to bottom of room-list to indicate more items in list */
+.room-list-blur {
+  height: 2.4rem;
+  /* -15px for scrollbar */
+  width: calc(100% - 15px);
+  position: absolute;
+  margin-top: -2.4rem;
```

Comment:
Please add a comment about where this calc comes from, both here and in the height.